includes/Story_Post_Type.php

```diff
@@ -523,7 +523,7 @@ public function get_editor_settings() {
 			'preview_nonce' => wp_create_nonce( 'post_preview_' . $story_id ),
 		];
 
-		$demo = isset( $_GET['web-stories-demo'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$is_demo = ( isset( $_GET['web-stories-demo'] ) && boolval( $_GET['web-stories-demo'] ) ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Should use `(bool) for type case, not `boolval`

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/5507/commits/d279dbf3036dbfc8045b3a66b8563a689c67e32c